src/ansible/roles/dns/templates/etc.dnsmasq.conf.j2

```diff
@@ -13,9 +13,12 @@ domain=test
 cache-size=0
 
 # These zones have their own DNS server
-{% for host in groups['ipa'] %}
-server=/{{ hostvars[host]['ansible_facts']['domain'] }}/{{ hostvars[host]['ansible_facts']['default_ipv4']['address'] }}
-{% endfor %}
+{% if 'master.ipa.test' in hostvars %}
+server=/ipa.test/{{ hostvars['master.ipa.test']['ansible_facts']['default_ipv4']['address'] }}
```

Comment:
Why You did not use the meta_domain variable I suggested?

Reply:
I did originally but, Justin brought up a good point about that variable being available outside of our ci.  I don't think we use these outside of that but currently, but, it was easier to just use the same format as the rest of the file which relies already on some hardcoded entries.

Reply:
Alternatively there is this code that might be used as base:
https://github.com/SSSD/sssd-ci-containers/blob/master/src/ansible/roles/ipa/tasks/main.yml#L13